### ISSUE TITLE
_Inspect_FscArgs does not exist

### ISSUE THREAD

**Issue by AndreasHassing** [ionide/ionide-vscode-fsharp#427]
Ionide fails to read my [`.fsproj`](https://github.com/AndreasHassing/microc-to-webassembly/blob/master/micro-wac/microwac.fsproj) file and therefore can't give proper autocompletion and type hints.

![image](https://cloud.githubusercontent.com/assets/639843/25658700/7e59b762-3004-11e7-8f11-ed1175698bcd.png)

The error encountered is: `error MSB4057: The target "_Inspect_FscArgs" does not exist in the project.`.

![image](https://cloud.githubusercontent.com/assets/639843/25658740/bbc4b39a-3004-11e7-88c6-15f9b789e2c8.png)

And more specifically, found in `F# Language Service` (added newlines for the sake of readability):
```
[13:30:57 DEBUG] RES (081) <- {project} in 690 ms: Kind={"error"}
Data="MSBuild failed with exitCode 1 Working Directory: '/Users/abn/microc-to-webassembly/micro-wac' Exe Path: 'dotnet'
Args: 'msbuild /Users/abn/microc-to-webassembly/micro-wac/microwac.fsproj
/p:SkipCompilerExecution=true /p:ProvideCommandLineArgs=true /p:CopyBuildOutputToOutputDirectory=false /p:UseCommonOutputDirectory=true
/t:_Inspect_FscArgs
/p:_Inspect_FscArgs_OutFile=/var/folders/rd/ytfxj23x2xvdpkpx195wlwxr0000gn/T/tmp3f2a37d2.tmp
/t:_Inspect_GetProjectReferences /p:_Inspect_GetProjectReferences_OutFile=/var/folders/rd/ytfxj23x2xvdpkpx195wlwxr0000gn/T/tmp768db84f.tmp
/t:_Inspect_GetProperties
/p:_Inspect_GetProperties_OutFile=/var/folders/rd/ytfxj23x2xvdpkpx195wlwxr0000gn/T/tmp176585b2.tmp
/nologo /verbosity:quiet'
```

I'm using Mono 4.8.1, dotnet cli 1.0.3, VSC 1.11.2 and Ionide 2.25.10 on a macOS Sierra 10.12.4.

I've got no clue what's going on. A Google search of the `_Inspect_FscArgs` keyword yielded a stunning: 0 results.

Keywords: ionide, namespace or module is not defined

**Comment by Krzysztof-Cieslak** [comment]
CC: @enricosada

**Comment by enricosada** [comment]
Problem is languages service ( FSAC ) is trying to parse your fsproj like it was of new sdk (.net coreclr sdk) because there is 15 in <Project

Error is https://github.com/fsharp/FsAutoComplete/blob/e8a41b2c922cd158b9b987d795f381fd829f28b7/src/FsAutoComplete.Core/Commands.fs#L122 
It should just check If contains "Sdk". 
The 15 was to discriminate the old dotnetcore core sdk rc3

**Comment by AndreasHassing** [comment]
Should I post the error on that project instead then? And thank you for the response, that makes sense.

**Comment by enricosada** [comment]
If you can, please just edit that file (online github is enough) and send as PR
Ionide will just use that in next release, right @Krzysztof-Cieslak ?
Sry wth mobile ATM, I cannot do it

**Comment by AndreasHassing** [comment]
Alright @enricosada, added a PR https://github.com/fsharp/FsAutoComplete/pull/163, also removed now redundant documentation, unsure if you want to keep it?

**Comment by enricosada** [comment]
Thx for removing doc!
Merged.
Now is @Krzysztof-Cieslak time :)

**Comment by Krzysztof-Cieslak** [comment]
Fix released in `2.25.11`
